src/CoreLib/client.fs

```diff
@@ -225,13 +206,20 @@ type internal ClientLauncher() =
                 null
         let listener = Listener.StartListener()
         let monTimer = ThreadPoolTimer.TimerWait ( fun _ -> sprintf "Task Queue Monitor Timer" ) listener.TaskQueue.MonitorTasks DeploymentSettings.MonitorIntervalTaskQueueInMs DeploymentSettings.MonitorIntervalTaskQueueInMs
+        
         // set authentication parameters for networking
+        Logger.Log( LogLevel.Info, (sprintf "Authentication parameters: pwd=%s keyfile=%s keyfilepwd=%s" pwd keyfile keyfilepwd ) )
```

Comment:
Do we want to log passwords directly in logfile (they will be visible in the clear)?